ci/docker/runtime_functions.sh

```diff
@@ -38,9 +38,30 @@ clean_repo() {
 build_jetson() {
     set -ex
     pushd .
-    mv make/crosscompile.jetson.mk make/config.mk
-    make -j$(nproc)
 
+    # Needed for FindCUDA.cmake module
+    export CUDA_TOOLKIT_ROOT=${CUDA_PATH}
+
+    cd /work/build
+    cmake \
+	-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE} \
+        -DCMAKE_CROSSCOMPILING=ON \
+        -DUSE_CUDA=ON \
+        -DUSE_CUDNN=ON \
+        -DUSE_OPENCV=OFF \
+        -DUSE_OPENMP=ON \
+        -DUSE_SSE=OFF \
+        -DCMAKE_C_FLAGS=-DMSHADOW_USE_SSE=0 \
```

Comment:
You probably also want to set -DMSHADOW_USE_F16C=0 similar to SSE, because Jetson would not support F16C as well

Reply:
@lebeg